src/main/java/analyzer/solutions/OptimalFormat.java

```diff
@@ -0,0 +1,62 @@
+package analyzer.solutions;
+
+import com.github.javaparser.ast.expr.*;
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.Node;
+import static com.github.javaparser.JavaParser.*;
+import java.io.File;
+import com.github.javaparser.JavaParser;
+import java.util.List;
+import com.github.javaparser.ast.visitor.VoidVisitorAdapter;
+import com.github.javaparser.ast.stmt.IfStmt;
+import com.github.javaparser.ast.stmt.ReturnStmt;
+import com.github.javaparser.ast.visitor.TreeVisitor;
+import java.util.ArrayList;
+
+
+public class OptimalFormat {
+
+    public static CompilationUnit getContent(String flname) throws Exception{
+        return JavaParser.parse(new File(flname));
+    }
+
+
+    // visits the method call expressions and adds them to a List
+    private static class OptionalMethodVisitor extends VoidVisitorAdapter<List<MethodCallExpr>> {
+        @Override
+        public void visit(MethodCallExpr n, List<MethodCallExpr> arg) {
+            arg.add(n);
+            super.visit(n, arg);
+        }
+    }
+
+    // Checks the return for the correct statement (optimal solution 1)
+    public static boolean checkReturn(List<ReturnStmt> returns){
+        if(returns.size() > 1){
```

Comment:
We should either check for multiple returns outside of this or inside of this class, but not both.

Also, we need a consistent answer on how to return suggestions (like "Can be refactored to use less return statements").